views.py

```diff
@@ -64,34 +65,37 @@ def download(request, path, *args, **kwargs):
         response.content = "<h1>You do not have permission to download this resource!</h1>"
```

Comment:
Since this download function is used by REST API, it would be better not to include the ```<h1>``` tag as part of the string.